tools/generate_tests.ml

```diff
@@ -0,0 +1,108 @@
+#!/usr/bin/env ocaml
```

Comment:
Why calling env here?

Reply:
more generic

Reply:
But you give no arguments to env except ocaml.

Reply:
but it allows me to not depend on the precise path to ocaml right?

Reply:
After the man page of env, it's just a way to set some environment variables before calling a command: env X=foo cmd.

Reply:
Yes, but it searches in the path for you, this way, if ocaml is installed in an opam prefix, env will get the opam one

Reply:
Ah I see. Right.